runtime/autoload/dist/ft.vim

```diff
@@ -532,6 +532,27 @@ export def FTm()
   endif
 enddef
 
+export def FTmake()
+  # Check if it is a Microsoft Makefile
+  if exists("b:make_microsoft")
+    unlet b:make_microsoft
```

Comment:
I think we can just use `:unlet!` instead of the `exists()` part. I'll include it with this tiny change.